Tasks/AzureWebAppSlotSwap/azurewebappslotswap.ts

```diff
@@ -0,0 +1,86 @@
+import tl = require('vsts-task-lib/task');
+import Q = require('q');
+import path = require('path');
+import httpClient = require('vso-node-api/HttpClient');
+
+var azureRmUtil = require ('./azurermutil.js');
+
+var httpObj = new httpClient.HttpClient(tl.getVariable("AZURE_HTTP_USER_AGENT"));
+
+var armUrl = 'https://management.azure.com/';
+var azureApiVersion = 'api-version=2015-08-01';
+
+
+async function run() {
+    try {
+        tl.setResourcePath(path.join( __dirname, 'task.json'));
+        var connectedServiceName = tl.getInput('ConnectedServiceName', true);
+        var webAppName: string = tl.getInput('WebAppName', true);
+        var resourceGroupName: string = tl.getInput('ResourceGroupName', true);
+        var source: string = tl.getInput('Source', true);
+        var destination: string = tl.getInput('Destination', true);
+        var preserveVnet: boolean = tl.getBoolInput('PreserveVnet', false);
+        var endPointAuthCreds = tl.getEndpointAuthorization(connectedServiceName, true);
+
+        var isSlotSwapSuccess = true;
+        var SPN = new Array();
```

Comment:
let SPN = <any>{}; ???
or better yet define an interface of these valid properties instead of any.